### PR TITLE
Add social share buttons

### PR DESCRIPTION
**Pull request by query-string**
This PR adds social buttons (Facebook, Twitter, Linkedin, Tumblr, Livejournal, Email, Print) requested in issue #3

### CODE REVIEW

**Review by FanaHOVA — APPROVED** [review]

File: source/layouts/layout.slim [hunk 1]
```diff
@@ -12,6 +12,7 @@ html
     - categories.each do |category|
       = feed_tag :atom, "#{config[:platform_url]}/categories/#{category.slug}.xml", title: "#{config[:platform_title]} – #{category.title} articles"
     = stylesheet_link_tag :blog
+    script src="//platform-api.sharethis.com/js/sharethis.js#property=58ea34c2b23bb10011d6ce9a&product=inline-share-buttons" type="text/javascript"
```
Comment (skatkov) [inline source/layouts/layout.slim]: can you bring this script to bottom please? maybe load it after page is loaded?

**Review by skatkov — APPROVED** [review]
This will work, but we can improve render times if we will delay loading of shareble buttons.

File: source/layouts/layout.slim (hunk 1 above)
Comment (query-string) [reply source/layouts/layout.slim]: @skatkov good point, thanks – I've blindly pasted the script as sharethis requested, but I'm sure it is possible to optimize loading a bit.

Comment (query-string) [reply source/layouts/layout.slim]: @skatkov delayed loading request is complete... As long as this script is the only bit of JS logic on the project I've deiced not to involve heavy third-party libraries (such as jQuery or domready.js) to detect page loading state and scripts requirement and just use a [piece of vanilla JS](https://github.com/saberespoder/sepcontent-v2/pull/21/files#diff-80ba1f9592234480fc349d018d11eb4cR1) for this purpose.

**Pull request merged**